kod/object/active/holder/room.kod

```diff
@@ -1773,6 +1773,22 @@ messages:
                  #new_col=Nth(new_pos,2));
          }
 
+         for i in plActive
+         {
+            each_obj = First(i);
+            if IsClass(each_obj,&Monster)
+            {
+               if NOT Send(what,@CheckPlayerFlag,#flag=PFLAG_ANONYMOUS)
```

Comment:
Seems like these checks should be done outside the outer loop, not on each iteration.